spec.html

```diff
@@ -30095,9 +30120,9 @@ <h1>String.prototype.toLowerCase ( )</h1>
         <emu-alg>
           1. Let _O_ be ? RequireObjectCoercible(*this* value).
           1. Let _S_ be ? ToString(_O_).
-          1. Let _cpList_ be a List containing in order the code points as defined in <emu-xref href="#sec-ecmascript-language-types-string-type"></emu-xref> of _S_, starting at the first element of _S_.
-          1. Let _cuList_ be a List where the elements are the result of toLowercase(_cpList_), according to the Unicode Default Case Conversion algorithm.
-          1. Let _L_ be the String value whose code units are the UTF16Encoding of the code points of _cuList_.
+          1. Let _sText_ be UTF16DecodeString(_S_).
+          1. Let _lowerText_ be the result of toLowercase(_sText_), according to the Unicode Default Case Conversion algorithm.
+          1. Let _L_ be ! UTF16Encode(_lowerText_).
```

Comment:
Is there a reason `UTF16Encode` is called with `!`, but `UTF16DecodeString` is not? I'm not asking that they be made consistent; I'll get to that as part of addressing https://github.com/tc39/ecma262/issues/1796, and both are technically allowed right now. But I want to make sure I'm not missing something.

Reply:
Per current conventions, I've added it throughout, and we can address it in #1796.